03.macos/05.solving-problems/07.icloud-private-relay/docs.en.md

```diff
@@ -0,0 +1,19 @@
+---
+title: 'iCloud Private Relay not working alongside AdGuard for Mac'
+taxonomy:
+    category:
+        - docs
+visible: true
+---
+
+Currently, AdGuard and iCloud Private Relay cannot work at the same time. 
```

Comment:
I'd suggest a different plan for this article:

1. Describe the problem: AdGuard and iCloud Private Relay cannot work at the same time
2. Describe the recommended solution: We suggest using a more traditional VPN service. For example, AdGuard VPN. (link to the website).
3. Describe the alternative solution via the advanced settings. Describe the drawbacks of that solution.

Reply:
@ameshkov Please check the updated diff for this file. The article flows much better now. Could you check the last sentence if the section **Alternative solution** for clarity?

Reply:
It makes sense to mention in the beginning, that AdGuard by default disables iCloud Private Relay.

Other than that, everything looks okay to me.